### PR TITLE
Added bio and put artwork to redirect to different pages, frontend (prettify)

### PR DESCRIPTION
**Pull request by mgomez03**
I added the bio but got rid of the status because I am working on a bug but I wanted to make sure that the front end stuff i am doing works before tomorrow.(specifically the gif) So The bio works and I changed the index file to add media to the homepage. I also added the artwork that directs to the homepage on every other page(top left corner) I'm sure that the jpg files will work, it's just the gif im worried about because I implemented that differently. It should look like this:
scrn shot

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Good job!. Are the img redirecting to those pages?

**Comment by mgomez03** [comment]
@joserenev Yes!

**Review by scoheleach — APPROVED** [review]
this looks so good!

File: src/main/webapp/WEB-INF/view/conversations.jsp [hunk 1]
```diff
@@ -25,7 +25,8 @@
 <body>
 
   <nav>
-    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a id="navTitle" href="/">
+    <img src = "https://drive.google.com/uc?id=1dG9V-sBNMS9hEivT4L-sn1M0m7RIn0Gr" width="160" height="80" /></a>
```
Comment (KevinWorkman) [inline src/main/webapp/WEB-INF/view/conversations.jsp:29]: You could consider putting these image files directly inside the project. Not a big deal though as long as this works!

**Review by KevinWorkman — COMMENTED** [review]
Do these changes leave the chat app in a state that's not broken? In other words, is this safe to push to the live site?

**Comment by mgomez03** [comment]
@KevinWorkman as far as i know everything works swimmingly! I've run it a few times and the functionality of everything else works just fine!

**Review by KevinWorkman — APPROVED** [review]
Okay cool! I'd maybe recommend giving it a more descriptive title, but other than that this LGTM.

**Comment by mgomez03** [comment]
Gotcha!

**Pull request merged**